src/header/header.scss

```diff
@@ -25,159 +25,105 @@
-  }
-
-  > .iui-right {
-    justify-content: flex-end;
+@mixin iui-page-header-zones {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
 
-    @media (prefers-reduced-motion: no-preference) {
-      .iui-user-icon {
-        &,
-        * {
-          transition: all $iui-speed-fast ease-out;
-        }
-      }
-    }
-  }
+@mixin iui-page-header-left {
+  @include iui-page-header-zones;
 
-  .iui-button {
+  > nav {
+    display: flex;
+    align-items: center;
     height: 100%;
-    border-radius: 0;
   }
 
-  .iui-divider {
-    width: 1px;
-    height: $iui-baseline * 2;
-    @include themed {
-      background-color: t(iui-color-background-4);
-    }
+  &,
+  > nav {
+    overflow: hidden;
+    box-sizing: content-box;
+    margin-bottom: -$iui-xs;
+    padding-bottom: $iui-xs;
   }
+}
 
-  &.iui-slim {
-    height: round($iui-baseline * 2.5) + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
-
-    > .iui-left {
-      .iui-header-logo {
-        > .iui-label {
-          font-size: $iui-font-size;
-        }
-
-        .iui-header-button-icon {
-          margin: 0 $iui-l + $iui-xs;
-        }
-      }
-
-      .iui-header-button-icon {
-        width: $iui-icons-default;
-        height: $iui-icons-default;
-        margin: 0 $iui-xs;
-        display: inline-flex;
-      }
-    }
-
-    > .iui-right {
-      .iui-user-icon {
-        @include iui-user-icon-size($size: $iui-icons-large, $font-size: $iui-font-size-small);
-      }
-    }
-  }
+@mixin iui-page-header-center {
+  @include iui-page-header-zones;
+}
 
-  @media (max-width: 768px) {
-    &,
-    &.iui-slim {
-      .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            @include iui-display-none;
-          }
-
-          .iui-header-button-icon + .iui-button-label {
-            margin-left: 0;
-          }
-        }
-      }
-    }
-  }
+@mixin iui-page-header-right {
+  @include iui-page-header-zones;
 
-  @media (max-width: 1024px) {
-    .iui-left {
-      .iui-header-logo > .iui-label {
-        display: none;
+  @media (prefers-reduced-motion: no-preference) {
+    .iui-user-icon {
+      &,
+      * {
+        transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out, font-size $iui-speed-fast ease-out;
       }
     }
   }
 }
+// #endregion Selection
 
-@mixin iui-header-button-icon($size: $iui-icons-large, $border-radius: $iui-border-radius) {
-  width: $size;
-  height: $size;
-  border-radius: $border-radius;
-  @media (prefers-reduced-motion: no-preference) {
-    transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out, margin $iui-speed-fast ease-out;
-  }
+// #region Header buttons
+@mixin iui-header-button-reset {
+  height: 100%;
+  border-radius: 0 !important;
 }
 
-@mixin iui-header-buttons {
-  .iui-header-button-icon {
-    @include iui-header-button-icon;
-  }
+@mixin iui-header-brand {
```

Comment:
Sure about the renaming? I liked logo..

Reply:
I like the renaming because `iui-header-brand` includes both the `icon` and `label`. Having a class name `iui-header-logo-icon` sounds redundant in my opinion